cclib/parser/turbomoleparser.py

```diff
@@ -208,6 +209,75 @@ def extract(self, inputfile, line):
             self.set_attribute('vibirs', vibirs)
             self.set_attribute('vibdisps', vibdisps)
 
+        # Parsing the scfenergies, scfvalues and scftargets from job.last file.
+        # scf convergence criterion : increment of total energy < .1000000D-05
+        #                  and increment of one-electron energy < .1000000D-02
+        #
+        # ...
+        # ...
+        #                                              current damping :  0.700
+        # ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     NORM[dD(SAO)]  TOL
+        #   1  -382.34543727790    -1396.8009423     570.56292464    0.000D+00 0.556D-09
+        #                            Exc =   -57.835278090846     N = 69.997494722    
+        #          max. resid. norm for Fia-block=  2.782D-05 for orbital     33a         
+        # ...
+        # ...
+        #                                              current damping :  0.750
+        # ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     NORM[dD(SAO)]  TOL
+        #   3  -382.34575357399    -1396.8009739     570.56263988    0.117D-03 0.319D-09
+        #                            Exc =   -57.835593208072     N = 69.999813370    
+        #          max. resid. norm for Fia-block=  7.932D-06 for orbital     33a         
+        #          max. resid. fock norm         =  8.105D-06 for orbital     33a         
+        #
+        # convergence criteria satisfied after  3 iterations
+        #
+        #
+        #                  ------------------------------------------ 
+        #                 |  total energy      =   -382.34575357399  |
+        #                  ------------------------------------------ 
+        #                 :  kinetic energy    =    375.67398458525  :
+        #                 :  potential energy  =   -758.01973815924  :
+        #                 :  virial theorem    =      1.98255043001  :
+        #                 :  wavefunction norm =      1.00000000000  :
+        #                  .......................................... 
+        if 'scf convergence criterion' in line:
+            total_energy_threshold = self.float(line.split()[-1])
+            one_electron_energy_threshold = self.float(next(inputfile).split()[-1])
+            scftargets = [total_energy_threshold, one_electron_energy_threshold]
+            if not hasattr(self, 'scftargets'):

```

Comment:
Do you think it would be possible to use `set_attribute` here? If not, are you able to add a new helper method that adds to an existing attribute?